.prettierignore

```diff
@@ -2,12 +2,14 @@
 build
 coverage
 
-**/volumes/*
-**/build/*
-**/dist/*
-**/tests/*
+**/volumes/**
+**/build/**
+**/dist/**
+**/tests/**
```

Comment:
I think we should also format tests files?

Reply:
removed tests on prettierignore